storage/redistore/peer_crud.go

```diff
@@ -0,0 +1,70 @@
+package redistore
```

Comment:
I initially named this as redis and renamed because because redigo/redis is also a package and they both derive in a very similar format (i.e redis.)